src/Resources/skeleton/doctrine/Repository.tpl.php

```diff
@@ -13,15 +13,31 @@ public function __construct(RegistryInterface $registry)
         parent::__construct($registry, <?= $entity_class_name ?>::class);
     }
 
+//    /**
+//     * @return <?= $entity_class_name ?>[] Returns an array of <?= $entity_class_name ?> objects
+//     */
     /*
-    public function findBySomething($value)
+    public function findByExampleField($value)
     {
         return $this->createQueryBuilder('<?= $entity_alias ?>')
-            ->where('<?= $entity_alias ?>.something = :value')->setParameter('value', $value)
+            ->andWhere('<?= $entity_alias ?>.exampleField = :val')
+            ->setParameter('val', $value)
             ->orderBy('<?= $entity_alias ?>.id', 'ASC')
             ->setMaxResults(10)
             ->getQuery()
-            ->getResult()
+            ->execute()
+        ;
+    }
+    */
+
+    /*
+    public function findOneBySomeField($value): <?= $entity_class_name."\n" ?>
```

Comment:
missing `?` for the return type. It MUST be a nullable one for a method return one or null result